### ISSUE TITLE
I cannot use RobotJS in my ElectronJS project

### ISSUE THREAD

**Issue by devliberte** [octalmage/robotjs#732]
Link to the project where I received an error: https://github.com/devliberte/electronrobotjs
I am trying to include the RobotJS library in my ElectronJS project. I have done everything as it should be and started the project. However, when I click the button, it gives an error. What could be the source of the problem? I would appreciate it if you could help me.
![gataso n](https://user-images.githubusercontent.com/126791600/224858720-dcebdcc6-5735-45ae-bf85-fc60507724f5.png)

**Comment by devliberte** [comment]
> if you use webpack， you can externals it as commonjs
I could not fully understand. Can you elaborate a little more?

**Comment by s1hofmann** [comment]
Given the error message and the fact that the repo you linked is just a plain Electron project without any bundler I’d simply assume that the robotjs build process failed.

The error message tells you that it can’t locate a `robotjs.node` bindings file, and since there are no prebuilds for robotjs, this bindings file would’ve to be built on your machine. If it fails, you can’t use it.

**Comment by LiuZhenyan-Wuzhong** [comment]
> Given the error message and the fact that the repo you linked is just a plain Electron project without any bundler I’d simply assume that the robotjs build process failed.
> 
> The error message tells you that it can’t locate a `robotjs.node` bindings file, and since there are no prebuilds for robotjs, this bindings file would’ve to be built on your machine. If it fails, you can’t use it.

Yes, there is no target file where the prebuild-install requests.
```
prebuild-install http request GET https://github.com/octalmage/robotjs/releases/download/v0.6.0/robotjs-v0.6.0-electron-v110-win32-x64.tar.gz
    prebuild-install http 404 https://github.com/octalmage/robotjs/releases/download/v0.6.0/robotjs-v0.6.0-electron-v110-win32-x64.tar.gz
    prebuild-install WARN install No prebuilt binaries found (target=22.3.2 runtime=electron arch=x64 
libc= platform=win32)
```

**Comment by octalmage** [comment]
Closing stale issues as part of a cleanup. If this is still an issue, please reopen and I'll take a look. Thanks!
